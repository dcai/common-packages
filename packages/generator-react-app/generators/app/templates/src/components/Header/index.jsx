import React from 'react';

const Menu = () => (
  <div className="navbar-menu">
    <div className="navbar-start">
      <a className="navbar-item">Home</a>
      <a className="navbar-item">Documentation</a>
    </div>

    <div className="navbar-end">
      <div className="navbar-item">
        <div className="buttons">
          <a className="button is-tab">Sign up</a>
          <a className="button is-tab">Log in</a>
        </div>
      </div>
    </div>
  </div>
);

const Brand = () => (
  <div className="navbar-brand">
    <a className="navbar-item" href="/">
      React Playground :)
    </a>
  </div>
);

const Header = () => (
  <nav
    className="navbar is-dark"
    role="navigation"
    aria-label="main navigation"
  >
    <Brand />
    <Menu />
  </nav>
);

export default Header;
