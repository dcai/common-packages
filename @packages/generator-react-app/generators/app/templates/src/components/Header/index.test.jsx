import React from 'react';
import enzyme from 'enzyme';
import Header from './index';

test('render a header', () => {
  const wrapper = enzyme.shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find('Brand').dive()).toMatchSnapshot();
});
