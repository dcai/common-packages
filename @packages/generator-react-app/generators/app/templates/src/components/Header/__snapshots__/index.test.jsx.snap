// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a header 1`] = `ShallowWrapper {}`;

exports[`render a header 2`] = `ShallowWrapper {}`;
